includes/Stories_Renderer/Renderer.php

```diff
@@ -519,6 +517,9 @@ class="<?php echo esc_attr( $single_story_classes ); ?>"
 			</div>
 			<?php
 		} else {
+			wp_enqueue_style( Embed_Base::STORY_PLAYER_HANDLE );
+			wp_enqueue_script( Embed_Base::STORY_PLAYER_HANDLE );
+			wp_enqueue_script( self::LIGHTBOX_SCRIPT_HANDLE );
```

Comment:
```suggestion
			$this->enqueue_style( Embed_Base::STORY_PLAYER_HANDLE );
			$this->enqueue_script( Embed_Base::STORY_PLAYER_HANDLE );
			$this->enqueue_script( self::LIGHTBOX_SCRIPT_HANDLE );
```

Reply:
same as [comment above](https://github.com/rtCamp/web-stories-wp/pull/188#discussion_r560760634)

Reply:
Yup, makes sense to keep as `wp_enqueue_style`